pkg/services/publicdashboards/models/errors.go

```diff
@@ -20,4 +20,6 @@ var (
 	ErrInvalidMaxDataPoints                = errutil.NewBase(errutil.StatusBadRequest, "publicdashboards.maxDataPoints", errutil.WithPublicMessage("maxDataPoints should be greater than 0"))
 	ErrInvalidTimeRange                    = errutil.NewBase(errutil.StatusBadRequest, "publicdashboards.invalidTimeRange", errutil.WithPublicMessage("Invalid time range"))
 	ErrInvalidShareType                    = errutil.NewBase(errutil.StatusBadRequest, "publicdashboards.invalidShareType", errutil.WithPublicMessage("Invalid share type"))
+
+	ErrPublicDashboardPaused = errutil.NewBase(errutil.StatusForbidden, "publicdashboards.paused", errutil.WithPublicMessage("Public dashboard paused"))
```

Comment:
Agree. gonna make those changes

Reply:
However, we still need to set the "Public dashboard paused" message, since the idea to that message is to be shown directly in the tooltip component, without any translation